src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -599,7 +599,9 @@ public function validateDecimal($attribute, $value, $parameters)
 
         $matches = [];
 
-        preg_match('/^[+-]?\d*.(\d*)$/', $value, $matches);
+        if (preg_match('/^[+-]?\d*\.?(\d*)$/', $value, $matches) !== 1) {
+            return false;
+        }
```

Comment:
The "dot" has been escaped here. I believe this was a mistake in the original PR.

The dot allows 1 of _any_ character. I believe it should only be a literal `.`.